libs/@blockprotocol/graph/src/graph-block-handler.ts

```diff
@@ -97,7 +98,9 @@ export class GraphBlockHandler
 
   // @todo automate creation of these methods from graph-service.json and types.ts
 
-  createEntity({ data }: { data?: CreateEntityData }) {
+  createEntity<
+    ValidProperties extends EntityPropertiesObject = EntityPropertiesObject,
+  >({ data }: { data?: CreateEntityData & { properties: ValidProperties } }) {
```

Comment:
Improvement: allow uses of the library to pass a generic to enforce a certain properties object (useful to get autocomplete and type safety when passing things in for an update, especially given keys are long URIs now)